delete-publication-from-scopus/src/main/java/no/sikt/nva/scopus/ScopusDeleteEventBody.java

```diff
@@ -0,0 +1,53 @@
+package no.sikt.nva.scopus;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Objects;
+
+import no.unit.nva.events.models.EventBody;
+import nva.commons.core.JacocoGenerated;
+
+public class ScopusDeleteEventBody implements EventBody {
+
+    public static final String TOPIC = "delete-scopus-identifier";
```

Comment:
Maybe we should have a topic "NvaFetchDoi.Scopus.Delete"

Reply:
Since the repo is called nva-fetch-doi that may fit. But actually Scopus has "nothing" todo with fetch DOI. So isn't NvaFetchDoi.Scopus a bit misleading? On the otherhand it is leading you to the right implementation....

Reply:
Will do!